checks/mysql

```diff
@@ -250,3 +250,132 @@ check_info['mysql.connections'] = {
     "group": "mysql_connections",
     "has_perfdata": True,
 }
+
+#
+# Galera Sync Status
+#
+
+def inventory_mysql_galerasync(parsed):
+    for instance, values in parsed.items():
+        if values.get('wsrep_provider', 'none') != 'none' and u'wsrep_local_state_comment' in values:
+            yield instance, {}
+
+def check_mysql_galerasync(item, _no_params, parsed):
+    if item in parsed:
+        values = parsed[item]
+        if u'wsrep_local_state_comment' in values:
+            if values[u'wsrep_local_state_comment'] == 'Synced':
```

Comment:
- Use readable strings; no internal strings: "WSREP local state comment: %s"
- If-block only checks the state, drag infotext out:
if STATEMENT:
  state = 2
else:
  state = 0
return state, "WSREP local state comment: %s" % VAL